WPFSKillTree/SkillTreeFiles/AscendancyClasses.cs

```diff
@@ -3,14 +3,9 @@
 
 namespace POESKillTree.SkillTreeFiles
 {
-    public class AscendancyClasses
+    public static class AscendancyClasses
```

Comment:
I'm not really a fan of static state, makes testing harder because of hidden dependencies.
Rather make those available through `ISkillTree` if necessary (that would be great, as it would lead to lower dependence of the build view model stuff on `SkillTree` than before your work).
Yes, SkillTree has a lot of static state. But that should be reworked at some point and it actually partly causes the lack of tests (e.g. I couldn't write tests for most of the tree generator stuff because of it).

(same thing with `BuildConverter`)